wn/web.py

```diff
@@ -336,4 +350,8 @@ async def synset(request):
     Route('/synsets', endpoint=all_synsets),
 ]
 
-app = Starlette(debug=True, routes=routes)
+app = Starlette(debug=True, routes=routes, exception_handlers={
+    HTTPException: http_exception_handler,
+    wn.Error: http_exception_handler,
+    Exception: http_exception_handler,
+})
```

Comment:
Maybe add this above at the end of the "Exception handlers" section of the code:

```python
exception_handlers = {
    HTTPException: http_exception_handler,
    wn.Error: http_exception_handler,
    Exception: http_exception_handler,
}
```

Then simplify this line as follows:

```suggestion
app = Starlette(debug=True, routes=routes, exception_handlers=exception_handlers)
```